src/libraries/Microsoft.XmlSerializer.Generator/src/Microsoft.XmlSerializer.Generator.csproj

```diff
@@ -7,8 +7,12 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
+
```

Comment:
This is a command line tool (`<OutputType>Exe</OutputType>`) - do we expect SGEN to ever be compiled as single file? I think we should just disable the analyzer for this instead of making changes to make it single file friendly

Reply:
Disabled SingleFileAnalyzer and reverted the changes